openreview/arr/arr.py

```diff
@@ -0,0 +1,418 @@
+        return self.venue.get_senior_area_chairs_id(number)
+
+    def get_ethics_chairs_id(self, number = None):
+        return self.venue.get_ethics_chairs_id(number)
+
+    def get_ethics_reviewers_id(self, number = None, anon=False):
+        return self.venue.get_ethics_reviewers_id(number, anon)
+
+    def get_publication_chairs_id(self):
+        return self.venue.get_publication_chairs_id()
+
+    def get_withdrawal_id(self, number = None):
+        return self.venue.get_withdrawal_id(number)
+
+    def get_withdrawn_id(self):
+        return self.venue.get_withdrawn_id()
+
+    def get_desk_rejection_id(self, number = None):
+        return self.venue.get_desk_rejection_id(number)
+
+    def get_desk_rejected_id(self):
+        return self.venue.get_desk_rejected_id()
+
+    def get_participants(self, number=None, with_program_chairs=False, with_authors=False):
+        return self.venue.get_participants(number, with_program_chairs, with_authors)
+
+    def get_submission_venue_id(self, submission_invitation_name=None):
+        return self.venue.get_submission_venue_id(submission_invitation_name)
+
+    def get_withdrawn_submission_venue_id(self, submission_invitation_name=None):
+        return self.venue.get_withdrawn_submission_venue_id(submission_invitation_name)
+
+    def get_desk_rejected_submission_venue_id(self, submission_invitation_name=None):
+        return self.venue.get_desk_rejected_submission_venue_id(submission_invitation_name)
+
+    def get_rejected_submission_venue_id(self, submission_invitation_name=None):
+        return self.venue.get_rejected_submission_venue_id(submission_invitation_name)
+
+    def get_submissions(self, venueid=None, accepted=False, sort='tmdate', details=None):
+        return self.venue.get_submissions(venueid, accepted, sort, details)
+
+    def expire_invitation(self, invitation_id):
+        return self.venue.expire_invitation(invitation_id)
+
+    def setup(self, program_chair_ids=[], publication_chairs_ids=[]):
+        setup_value = self.venue.setup(program_chair_ids, publication_chairs_ids)
+        setup_arr_invitations(self.invitation_builder)
+        return setup_value
+
+    def set_impersonators(self, impersonators):
+        return self.venue.set_impersonators(impersonators)
+
+    # For stage invitations, pass value to inner venue objects
+    def create_submission_stage(self):
+        self.venue.submission_stage = self.submission_stage
+        stage_value = self.venue.create_submission_stage()
+        invitation = self.client.get_invitation(self.get_submission_id())
+        invitation.preprocess = self.invitation_builder.get_process_content('process/submission_preprocess.py')
+        invitation.process = invitation.process + self.invitation_builder.get_process_content('process/submission_process_extension.py')
+        invitation.signatures=['~Super_User1']
+        self.invitation_builder.save_invitation(invitation)
```

Comment:
instead of saving the whole invitation again, can we just post an edit using the meta invitation that modified the `process`, `preprocess` and `signatures` fields?

Reply:
I'm thinking that using an edit with only the changes that ARR is overriding is easy to track.